package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "start": "NODE_ENV=production ./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
     "start:dev": "nodemon --config nodemon.json",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint": "eslint . --ext .ts",
```

Comment:
If we ever introduce some custom js in future we'll need to re-add this, but for now we can just stick with .ts